tools: abort CQ session when landing several commits (#40577)
nodejs/node: doc/guides/commit-queue.md

      3. Close the PR
      4. Go to next PR in the queue

> To make the Commit Queue squash all the commits on a pull request in the first

Reviewer: ```suggestion
To make the Commit Queue squash all the commits of a pull request into the first
```
